Mathlib/Tactic/Linter/Style.lean

```diff
@@ -55,7 +56,16 @@ def setOptionLinter : Linter where run := withSetOptionIn fun stx => do
     if (← MonadState.get).messages.hasErrors then
       return
     if let some head := stx.find? is_set_option then
-      if let some name := parse_set_option head then
+      if let some (name, val) := parse_set_option head then
+        -- xxx: how to make this check typed? I'd like to check for a `str variant with content
+        -- "true"...
```

Comment:
You need to patch Lean core to add a `withAntiquot` on `Lean.Parser.Command.optionValue`.